tests/test_workspace_tabs_null_bubble.py

```
@@ -0,0 +1,81 @@
+
+        self.workspace_path = ws_dir
+
+    def tearDown(self):
+        self.tmp.cleanup()
+
+    def test_null_bubble_row_is_skipped_without_exception(self):
+        """assemble_workspace_tabs must not raise when a bubble row has NULL value."""
+        from services.workspace_tabs import assemble_workspace_tabs
+
+        try:
+            payload, status = assemble_workspace_tabs(
```

Ruff (0.15.12)

[warning] Unpacked variable `payload` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)